ops/ansible/playbooks-healthapt/files/client_data_server_dpc_test_certificate.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
+MIIEsDCCApigAwIBAgIEXW++/zANBgkqhkiG9w0BAQsFADAaMRgwFgYDVQQDDA9k
+cGMtdGVzdC1jbGllbnQwHhcNMTkwOTA0MTM0MTE5WhcNMjkwOTAxMTM0MTE5WjAa
+MRgwFgYDVQQDDA9kcGMtdGVzdC1jbGllbnQwggIiMA0GCSqGSIb3DQEBAQUAA4IC
+DwAwggIKAoICAQCVXNdpD3jvmtPZI8BB65udAJaVNMSLcQsz6HTxZXDrqJLYjhjA
+9xQBtHV90Dvicyjeh1t1zcpc5RPLO7y0YU7Kc7CQtk6yYFG3czSmA/yb9zJxXgoS
+p198r2tbi2VbIcYYWTaz8kIz52DZ1WPxv3Pu8wMt6yBbUrvfMiGcRSQ+W3ziTde7
+wC5U07W4RH/87hI5lm0yFq73ysMsxNTlZvmCfen7kE2mTbuh686KQXxlL3OJrmwp
+sReo9y1DHaiLlcjeHU/v0gXleAxw1Bb9icKwYqEkNEjjY6rGU46pH5pBAeCX613l
+/TRUypXd0JL+zqa0k5UDOaA/Szvk2huPgmtDxAMxXo63KnKEkMSAY9VWcSarsqXu
+1nIp0X9nCAqRkl/ElaDLbDufENp6WlQHo8hib9+Qgc+Yp+n5cBFVgbL/uJnL0+mF
+VHyC6/2nP/AsnHWNgeWRxrLc9FyChIYmVsy/42yNIyoM5tYWwRBUT0JPz3BIsl3S
+rnEFGVto8MQOLP5wCX0fo7bh8TBAPjqEv1WG11ntIIxsVGPBdq82eXlmWWYNmuSG
+0RSwrU8svgiXnrMU2cGFnrG8adlTZ87llscJ4aHqSlw0vKCuA7zSURaSI1AsIzlP
+H1kGESz+7unHYrkV+ql8SVLMMc9AeWV8wn5Dfw+vaM50/fCnKpUWjsTUxwIDAQAB
+MA0GCSqGSIb3DQEBCwUAA4ICAQBKrMo4elAIq+GHF6EX9hnx2ORSQXNwu9QzCuqh
+ABQ6L/dzrABS0EwiZsCDJSHOFT+Bz5jyzeVCbWIr5OHwFW6kvEshsWGTJLQ9/kwh
+yN5OSRGerVLma66hGUtx0PnGOXfPrEpfureOHFOzoUoW9hahhK0Hgu+TXiKpRWoo
++ePk+3Prl6TIb4WniUAxyUHw88Plt0lav8u4zt9xmA4HqHAtP+JVu9DkByb4WX1I
+fdt/jYszIeiY30240LQFkUbHBEsbPtnVEtByw0CjgBrYmSi8+l+KrSmArtoUs4Kg
+pT0K9HYy6C1Vr3/29YOxtatQjAhQk2arKZeBw9XRcoVi/db0gFS0/O90EWIp3g8R
+5il8kUXk43SLpk2JhqaCK50LX2HYkqwrqbzQpSGeGen/GO9y8Im9zz2o0r+IwxNU
+XqOk3QylfbBieKi1lAH61D/8QmpGNEfHbIJQj6wpAvp9FrLstDdBOomIoUE1nMVy
+T5c9x7wxCs1yj3YYEL7qEix91xTD6Ki4yR4o9Vg8iUgNybHpFT7z+365t6SNb2/f
+o6M0JaQMIU3CYcVAoAkY/EdcufP2BU/nBCnCZNfO0U034Liin6GRIAeXr67uIiET
+apcmXtWUugZb7N50/5VYtonOOHatHCrWfTFI/aGHczgSnSEphT+UHCK61i/jqF2P
+aa+UmA==
+-----END CERTIFICATE-----
```

Comment:
```
$ openssl x509 -text -in ops/ansible/playbooks-healthapt/files/client_data_server_dpc_test_certificate.pem
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 1567604479 (0x5d6fbeff)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=dpc-test-client
        Validity
            Not Before: Sep  4 13:41:19 2019 GMT
            Not After : Sep  1 13:41:19 2029 GMT
        Subject: CN=dpc-test-client
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:95:5c:d7:69:0f:78:ef:9a:d3:d9:23:c0:41:eb:
                    9b:9d:00:96:95:34:c4:8b:71:0b:33:e8:74:f1:65:
                    70:eb:a8:92:d8:8e:18:c0:f7:14:01:b4:75:7d:d0:
                    3b:e2:73:28:de:87:5b:75:cd:ca:5c:e5:13:cb:3b:
                    bc:b4:61:4e:ca:73:b0:90:b6:4e:b2:60:51:b7:73:
                    34:a6:03:fc:9b:f7:32:71:5e:0a:12:a7:5f:7c:af:
                    6b:5b:8b:65:5b:21:c6:18:59:36:b3:f2:42:33:e7:
                    60:d9:d5:63:f1:bf:73:ee:f3:03:2d:eb:20:5b:52:
                    bb:df:32:21:9c:45:24:3e:5b:7c:e2:4d:d7:bb:c0:
                    2e:54:d3:b5:b8:44:7f:fc:ee:12:39:96:6d:32:16:
                    ae:f7:ca:c3:2c:c4:d4:e5:66:f9:82:7d:e9:fb:90:
                    4d:a6:4d:bb:a1:eb:ce:8a:41:7c:65:2f:73:89:ae:
                    6c:29:b1:17:a8:f7:2d:43:1d:a8:8b:95:c8:de:1d:
                    4f:ef:d2:05:e5:78:0c:70:d4:16:fd:89:c2:b0:62:
                    a1:24:34:48:e3:63:aa:c6:53:8e:a9:1f:9a:41:01:
                    e0:97:eb:5d:e5:fd:34:54:ca:95:dd:d0:92:fe:ce:
                    a6:b4:93:95:03:39:a0:3f:4b:3b:e4:da:1b:8f:82:
                    6b:43:c4:03:31:5e:8e:b7:2a:72:84:90:c4:80:63:
                    d5:56:71:26:ab:b2:a5:ee:d6:72:29:d1:7f:67:08:
                    0a:91:92:5f:c4:95:a0:cb:6c:3b:9f:10:da:7a:5a:
                    54:07:a3:c8:62:6f:df:90:81:cf:98:a7:e9:f9:70:
                    11:55:81:b2:ff:b8:99:cb:d3:e9:85:54:7c:82:eb:
                    fd:a7:3f:f0:2c:9c:75:8d:81:e5:91:c6:b2:dc:f4:
                    5c:82:84:86:26:56:cc:bf:e3:6c:8d:23:2a:0c:e6:
                    d6:16:c1:10:54:4f:42:4f:cf:70:48:b2:5d:d2:ae:
                    71:05:19:5b:68:f0:c4:0e:2c:fe:70:09:7d:1f:a3:
                    b6:e1:f1:30:40:3e:3a:84:bf:55:86:d7:59:ed:20:
                    8c:6c:54:63:c1:76:af:36:79:79:66:59:66:0d:9a:
                    e4:86:d1:14:b0:ad:4f:2c:be:08:97:9e:b3:14:d9:
                    c1:85:9e:b1:bc:69:d9:53:67:ce:e5:96:c7:09:e1:
                    a1:ea:4a:5c:34:bc:a0:ae:03:bc:d2:51:16:92:23:
                    50:2c:23:39:4f:1f:59:06:11:2c:fe:ee:e9:c7:62:
                    b9:15:fa:a9:7c:49:52:cc:31:cf:40:79:65:7c:c2:
                    7e:43:7f:0f:af:68:ce:74:fd:f0:a7:2a:95:16:8e:
                    c4:d4:c7
                Exponent: 65537 (0x10001)
    Signature Algorithm: sha256WithRSAEncryption
         4a:ac:ca:38:7a:50:08:ab:e1:87:17:a1:17:f6:19:f1:d8:e4:
...
```

Things I'm checking:

1. There's no private key material here, right? Correct.
1. Is the cert different from any other certs from the same project (just watching out for oopses)? Yes.
1. Is its Subject unique? Yes.
1. Does it have a reasonably useful/informative Subject? Yes.
1. Does it have a reasonable key size (>= 2048)? Yes.